io.github.eminfedar.vaktisalah-gtk-rs.json

```
@@ -0,0 +1,86 @@
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "44",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "vaktisalah-gtk-rs",
+    "finish-args": [
+        "--share=network",
+        "--socket=pulseaudio",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+
+    "modules": [
+        {
+            "name": "libsass",
+            "buildsystem":"meson",
+            "builddir": true,
+            "sources": [
+                {
+                    "type":"git",
+                    "url":"https://github.com/lazka/libsass.git",
+                    "commit": "aac79dccd3c8f7e8f22125f87a119f3b1ee9d487"
+                }
+            ]
+        },
+        {
+            "name": "sassc",
+            "buildsystem":"meson",
+            "builddir": true,
+            "sources": [
+                {
+                    "type":"git",
+                    "url":"https://github.com/lazka/sassc.git",
+                    "commit": "a1950c2d95ea4c051feb90bb1f43559fbb54bf36"
+                }
+            ]
+        },
+        {
+            "name": "blueprint-compiler",
+            "buildsystem": "meson",
+            "sources": [
+              {
+                "type": "git",
+                "url": "https://gitlab.gnome.org/jwestman/blueprint-compiler",
+                "tag": "v0.6.0"
+              }
+            ],
+            "cleanup": ["*"]
+        },
+        {
+            "name": "vaktisalah-gtk-rs",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/eminfedar/vaktisalah-gtk-rs",
+                    "tag": "v1.0.1"
+                },
+                "cargo-sources.json"
+            ],
+            "build-options": {
+                "append-path" : "/usr/lib/sdk/rust-stable/bin",
+                "env" : {
+                    "CARGO_HOME" : "/run/build/vaktisalah-gtk-rs/cargo",
+                    "LOCALIZATION_DIR": "/run/build/vaktisalah-gtk-rs/localization"
+                }
+            },
+            "build-commands": [
+                "cargo --offline fetch --manifest-path Cargo.toml",
+                "cargo --offline build --release",
+                "install -Dm755 ./target/release/vaktisalah-gtk-rs -t /app/bin/"
+            ],
+            "post-install": [
+                "install -Dm644 export/io.github.eminfedar.vaktisalah-gtk-rs.svg /app/share/icons/hicolor/scalable/apps/io.github.eminfedar.vaktisalah-gtk-rs.svg",
+                "install -Dm644 export/io.github.eminfedar.vaktisalah-gtk-rs.desktop /app/share/applications/io.github.eminfedar.vaktisalah-gtk-rs.desktop",
+                "install -Dm644 export/io.github.eminfedar.vaktisalah-gtk-rs.xml /app/share/metainfo/io.github.eminfedar.vaktisalah-gtk-rs.xml"
```

```suggestion
                "install -Dm644 export/io.github.eminfedar.vaktisalah-gtk-rs.xml /app/share/metainfo/io.github.eminfedar.vaktisalah-gtk-rs.metainfo.xml"
```

Ideally the name upstream could be changed as it is incorrect.

---

and the original was fine.